upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -265,6 +265,29 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		}
 	}
 
+	{
+		// Adding the kubelet-api-admin binding: this is required when switching to webhook authorization on the kubelet
+		// docs: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#other-component-roles
+		// issue: https://github.com/kubernetes/kops/issues/5176
+		key := "rbac.addons.k8s.io"
+		version := "v0.0.1"
+
+		{
```

Comment:
You could possibly group this in with the other rbac addon block above, granting the kubelet user access to nodes (within the `enableRBACAddon` check). Both of these changes (not adding the CRB by default) look to have been introduced at the same time in k8s 1.8.

Reply:
That one is dependent of the use of a node-authorizer though ... as when the node-authorizer is enabled we remove the default mapping in the block